packages/expo-notifications/ios/EXNotifications/Notifications/EXNotificationObjcWrapper.m

```diff
@@ -0,0 +1,30 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+
+#import "EXNotificationObjcWrapper.h"
+
+@implementation EXNotificationObjcWrapper
+
+/**
+ Wrapper that allows Swift code to execute Objective-C code that could throw an NSException, which is not catchable by Swift.
+ */
++ (BOOL)tryExecute:(nonnull void(NS_NOESCAPE^)(void))tryBlock error:(__autoreleasing NSError * _Nullable * _Nullable)error {
```

Comment:
Maybe you can use this: https://github.com/expo/expo/blob/4c9c62752347568b5174cd537d3de15d03f58b65/packages/expo-modules-core/ios/Legacy/EXUtilities.m#L218-L228

Reply:
Thanks! I'll try that